build.gradle

```diff
@@ -68,6 +69,7 @@ ext.libraries = [
         mantisShaded   : "io.mantisrx:mantis-shaded:1.3.+",
         mockitoAll     : "org.mockito:mockito-all:${versions.mockito}",
         mockitoCore    : "org.mockito:mockito-core:${versions.mockito}",
+        mockitoCore3   : "org.mockito:mockito-core:${versions.mockito3}",
```

Comment:
I tried it briefly and there are quite some breaks in the current code when switching to mockito3. This could be a backlog work item. Tracked at #184